salt/_pillar/metalk8s_nodes.py

```diff
@@ -50,13 +53,100 @@ def node_info(node, ca_minion):
     return result
 
 
+def get_cluster_version(api_client):
+    coreV1 = kubernetes.client.CoreV1Api(api_client=api_client)
```

Comment:
Is it possible that this variable could be a `None` value for example if there is no connection? Or are we sure we'll always have a proper object?

Reply:
We'll always have an object, no worries to be had here.